ocean_lib/ocean/ocean_assets.py

```diff
@@ -171,9 +170,10 @@ def create(
             "dataset",
             "algorithm",
         ), f"Invalid/unsupported asset type {asset_type}"
-        if not plecos.is_valid_dict_local(metadata_copy):
-            errors = plecos.list_errors_dict_local(metadata_copy)
-            msg = f"Metadata has validation errors: {errors}"
+
+        validation_result, validation_errors = self.validate(metadata)
```

Comment:
Before, the call went from ocean.py -> plecos.

Now it goes from ocean.py -> `aquarius.validate_metadata()`. But aquarius itself uses ocean.py. So ocean.py and aquarius are circularly dependent on each other. Python doesn't complain because one direction is hidden under the REST API. It is a circular dependency nonetheless.

We shouldn't have circular dependencies.

Why not just move the functionality that's inside `aquarius.validate_metadata()` into ocean.py?